locale/de/LC_MESSAGES/django.po

```diff
@@ -9610,7 +9610,7 @@ msgstr ""
 "\n"
 "An: Bundesministerium des Innern\n"
 "\n"
-"Sehr geehrte Damen und Herren,\n"
+"Sehr geehrte Damen bis Herren,\n"
```

Comment:
I guess agencies will be confused by that…

Reply:
I don't think the confusion would last very long and the alternatives are more invasive to the "flow". E.g. "Sehr geehrte Menschen" may even be read as derogatory, a mere "Hallo" seems clearly too informal to me. Also when it causes confusion, that probably is for expanding the recipients horizont to people beyond the gender binary and therefor would be considered a feature, not a bug. I am however open to suggestions.

Reply:
`Sehr geehrte Alle,\n?`

Reply:
> `Sehr geehrte Alle,\n?`

Sounds good to me, good idea! :)

Reply:
How about `Guten Tag`?

```suggestion
"Guten Tag,\n"
```